package.json

```diff
@@ -21,12 +21,23 @@
   },
   "devDependencies": {
     "@eslint/js": "^9.0.0",
+    "@types/chai": "^4.3.14",
     "@types/express": "^4.17.21",
     "@types/express-session": "^1.18.0",
+    "@types/mocha": "^10.0.7",
     "@types/node": "^20.11.30",
     "@types/nunjucks": "^3.2.6",
+    "@types/selenium-webdriver": "^4.1.22",
+    "@types/sinon": "^17.0.3",
+    "axios-mock-adapter": "^1.21.4",
+    "chai": "^4.3.7",
+    "chromedriver": "^125.0.2",
     "eslint": "^8.57.0",
+    "mocha": "^10.2.0",
     "nodemon": "^3.1.0",
+    "sass": "^1.75.0",
+    "selenium-webdriver": "^4.9.0",
+    "sinon": "^17.0.1",
```

Comment:
are you sure you need all these dependencies for your PR?

Reply:
Whoops no, removed unused ones